### ISSUE TITLE
Build problems

### ISSUE THREAD

**Issue by SSP6904** [copy/v86#909]
Hi all!

I'm using the docker method to build v86, but the files "v86_all.js" and "libv86.js" always get deleted by the build commands inside my container. I have no idea why it does it. It also complains of an "unused label" or "variable" warning.

Is something wrong with my install or is it just the source code itself?

**Comment by spetterman66** [comment]
1. get github codespaces vps (free for any github acc, https://github.com/codespaces)
2. use copy/v86 or your own personal fork of it to put inside the vps
3. sudo su
4. run:
        export DEBIAN_FRONTEND=noninteractive && \
        dpkg --add-architecture i386 && \
        apt-get update -qq && \
        apt-get install -y nodejs nasm gdb unzip p7zip-full openjdk-8-jre wget python python3 qemu-system-x86 git-core build-essential libc6-dev-i386-cross libc6-dev-i386 clang curl time && \
        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
        export PATH="$HOME/.cargo/bin:$PATH" && \
        rustup toolchain install stable && \
        rustup target add wasm32-unknown-unknown && \
        rustup component add rustfmt-preview
6. wait for it to finish
7. run `make all`
8. when it's done, install `apache`
9. run the apache server: `service apache start`
10. `cp -r build /var/www/html`
11. `cp -r index.html v86.css /var/www/html`
12. go to ports inside the vscode
13. forward port 80
14. open the url
15. then you got selfhosted v86 :-)

**Comment by SSP6904** [comment]
I have got the method that @spetterman66 said worked.

I'm also wondering if the same steps are for the debug version since that one helps me out with problems regarding the VM.

**Comment by spetterman66** [comment]
yes, just replace `make all` with `make` to get the debug version, then copy the debug.html to `/var/www/html`. go to (your url).github.dev/debug.html and you're done :D
